bin/.travis/test.sh

```diff
@@ -78,18 +78,9 @@ docker run -ti --rm \
 printf "\nVersion and module information about php build\n"
 docker run -ti --rm ez_php:latest-dev bash -c "php -v; php -m"
 
-
 printf "\Integration: Behat testing on ez_php:latest and ez_php:latest-dev with eZ Platform\n"
 cd volumes/ezplatform
 
-# Tag image alias for what (exactly) eZ Platform is currently using in order to be able to test across branches
-## As we don't want it to pull in remote, but rather use what we just built here
-## NOTE: On larger changes to this images, we would need to pull in custom branches with adaptions on Platform side as well as below
-## TODO: The tag aliases here will not be needed once eZ Platform uses PHP_IMAGE variable within Dockerfile's from section
-docker tag ez_php:latest "ezsystems/php:7.2-v1"
-docker tag ez_php:latest "ezsystems/php:7.1-v1"
-docker tag ez_php:latest "ezsystems/php:7.0-v1"
```

Comment:
Not needed anymore as the configurable injection of this is now part of both 1.13 and 2.x releases:
https://github.com/ezsystems/ezplatform/commit/cb6794856afa0e00a3ad727e0b71e3b0b865a786